modules/tftp/tftp_plugin.rb

```diff
@@ -9,6 +9,9 @@ class Plugin < ::Proxy::Plugin
                      :verify_server_cert => true
     validate :verify_server_cert, boolean: true
 
+    # Expose automatic iso handling capability
+    capability 'extraction'
```

Comment:
You could make this dynamic and see if `isoinfo` is in the path. Upside is that it won't fail at runtime, downside is that you need to refresh the features from Foreman after installing `isofind`. For that downside we can compensate by handling the installation either in packaging or in the installer.

Reply:
Sounds like a good idea!
When you talk about making it dynamic, should I take the BMC plugin as an example and how it's done here for the IPMI part?
https://github.com/theforeman/smart-proxy/blob/a48da738dd0f41cb28c7042e5f2773a8d9a29293/modules/bmc/bmc_plugin.rb#L9-L14

I would suggest doing a similar implementation which checks the availability of `isoinfo` in this file:
https://github.com/theforeman/smart-proxy/blob/a48da738dd0f41cb28c7042e5f2773a8d9a29293/modules/tftp/tftp.rb#L1-L2

What do you think? Or should I create a similar class as it's done for IPMI instead of using the "base" class `tftp.rb`?

Reply:
> I would suggest doing a similar implementation which checks the availability of `isoinfo` in this file:

This indeed